.github/workflows/production_pipeline.yml

```diff
@@ -0,0 +1,133 @@
+name: 'Production CI/CD Pipeline'
+
+on:
+  release:
+    types: [released]
+  workflow_dispatch:
+
+concurrency: 'production'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:13.5
+        env:
+          POSTGRES_PASSWORD: password
+          POSTGRES_USER: postgres
+          POSTGRES_DB: data_collector
```

Comment:
I'm assuming we don't need to hide these?

Reply:
Not really, they're only used within this container to run the test stage, they're not copied to prod environment or anything like that